### PR TITLE
[SMALLFIX]@Nullable annotation in alluxio.master.file.meta.getBucketContaining 

### PR DESCRIPTION
**Pull request by anwaroff**
just want to make a pull request to alluxio without anything new

### CODE REVIEW

**Comment by pNswordsman** [comment]
Please following the tutorial we provided to finish your pr. This one is invalid!!!!

**Comment by apc999** [comment]
@anwaroff  you are definitely welcome to improve the code documentation. Could you let us know which part you want to improve? The current patch does not look complete

**Comment by apc999** [comment]
@anwaroff this change is unrelated to the title. I am not sure what is the goal of this PR, can you update the description? Otherwise, I may close the PR

**Comment by anwaroff** [comment]
i am trying to fix the problem please do not close it i will do it nicely soon. thank you On 2018-06-08 01:57 , Bin Fan Wrote: @anwaroff this change is unrelated to the title. I am not sure what is the goal of this PR, can you update the description? Otherwise, I may close the PR — You are receiving this because you were mentioned. Reply to this email directly, view it on GitHub, or mute the thread.

File: assembly/README.md [hunk 1]
```diff
@@ -25,3 +25,17 @@ should be included.
 
 For more details on how to use the generated tar, go
 [to the docs](http://alluxio.org/documentation/Running-Alluxio-Locally.html).
+
```
Comment (apc999) [inline assembly/README.md]: can you remove this added text?

File: core/server/master/src/main/java/alluxio/master/file/meta/TtlBucketList.java [hunk 2]
```diff
@@ -45,6 +45,12 @@ public TtlBucketList() {
    * @param inode the inode to be contained
    * @return the bucket containing the inode, or null if no such bucket exists
    */
+ /* private TtlBucket getBucketContaining(Inode<?> inode) {
```
Comment (apc999) [inline core/server/master/src/main/java/alluxio/master/file/meta/TtlBucketList.java]: remove line 48 to 52?

**Pull request closed without merging**